tests/table/tableView.test.ts

```diff
@@ -511,4 +511,265 @@ describe("Table view", () => {
             expect(tableView.tablePlaces.place1().Card4Hightlighted()).toEqual(true);
         });
     });
+    describe("Verify game event notifications", () => {
+        beforeAll(() => {
+            GameActionsQueue.waitDisabled = true;
+            GameActionsQueue.drainQueuePause = 0;
+            debugSettings.tableView.trace = false;
+        });
+        afterAll(() => {
+            GameActionsQueue.waitDisabled = false;
+            GameActionsQueue.drainQueuePause = 100;
+        });
+        it("Verify preflop", async () => {
+            const winner: GameWinnerModel[] = [
+                {
+                    Amount: 100,
+                    Pot: 1,
+                    CardsDescription: "",
+                    PlayerId: 1
+                }
+            ]
+            const tableView = new TableView(1, {
+                TableId: 1,
+                TableName: "",
+                BigBlind: 200,
+                SmallBlind: 100,
+                CurrencyId: 1,
+                HandsPerHour: 0,
+                AveragePotSize: 0,
+                JoinedPlayers: 2,
+                MaxPlayers: 8,
+                PotLimitType: 2,
+            });
+            const players: GamePlayerStartInformation[] = [{
+                PlayerId: 1,
+                Money: 10000,
+            }, {
+                PlayerId: 2,
+                Money: 10000,
+            }];
+            const actions: GameActionStartInformation[] = [];
+            login("player1");
+            loginId(1);
+            const tableSatusPlayers = [
+                getSeatPlayer(1, 10000),
+                getSeatPlayer(2, 10000),
+            ];
+            let counter = 0;
+            tableView.onPlayerCardsDealed.add(() => { counter = counter + 1 }, this);
```

Comment:
You should use different kind of counters here, otherwise you could not really distinguish between what kind of event happens.